keps/sig-storage/2268-non-graceful-shutdown/kep.yaml

```diff
@@ -0,0 +1,36 @@
+title: non graceful shutdown
+kep-number: 2264
+authors:
+  - "@yastij"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-node
+status: implemented
+creation-date: 2019-06-26
+reviewers:
+  - "@xing-yang"
+  - "@andrewsykim"
+approvers:
+  - "@saad-ali"
+  - "@thockin"
+prr-approvers:
```

Comment:
Please don't forget to add prr approver here as well.

Reply:
Done